conflate/conflate.py

```diff
@@ -625,6 +628,8 @@ def format_change(before, after, ref):
                     else:
                         props['tags_changed.{}'.format(k)] = '{} -> {}'.format(v0, v1)
             props['marker-color'] = MARKER_COLORS[marker_action]
+            if ref and ref.remarks:
+                props['__remarks__'] = ref.remarks
```

Comment:
I think `__` is excessive: just use `remarks` please.

Reply:
A quick search on overpass tells me, that there are (at least) three nodes in OSM which possess a tag named "remarks" (and a few more with "remark"). I think in order to ensure collisions, we should stick to a highly unlikely name. I agree, that using a `__` is a bit on the overly cautious side, but using plain `remarks` seems too lax to me. What about `_remarks`, seems a good compromise between "clash-freeness" and not too long?

Reply:
Tags are prefixed with `tags.`, `tags_changed.` etc.

Reply:
True, forgot about that. Thank you for the reminder, I'll change it accordingly!